src/content/en/2020/markup.md

```diff
@@ -1059,7 +1038,7 @@ When we built our tests we didn't check for the presence of images, but only loo
     <thead>
       <tr>
         <th>Favicon format</th>
-        <th>Occurrences</th>
+        <th>Pages</th>
```

Comment:
Same general problem as mentioned before and no, “pages” would be inaccurate.

Reply:
Think you might be right but then Pages (%) is also inaccurate?

![image](https://user-images.githubusercontent.com/10931297/103458355-0473c700-4cff-11eb-9cbe-d4332013215b.png)

Reply:
Same here, one result per page, so pages and % pages is accurate. I think in this case it will pick the first matching tag on the page.